keyboards/bandominedoni/keymaps/default/keymap.c

```diff
@@ -0,0 +1,118 @@
+/* Copyright 2021 3araht
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines names for use in layer keycodes and the keymap
+enum layer_names {
+#ifndef PEDAL_NORMALLY_CLOSED
+    _OPEN,
+#endif
+    _CLOSE,
+#ifdef PEDAL_NORMALLY_CLOSED
+    _OPEN,
+#endif
+    _QWERTY,
+    _MISC,
+    _FN
+};
+
+// Alias layout macros that expand groups of keys.
+#define LAYOUT_wrapper(...) LAYOUT(__VA_ARGS__)
```

Comment:
```suggestion
```

Reply:
@drashna
Same here. LAYOUT_wrapper is useful.
May I ask the background of it?